project/OSGi.scala

```diff
@@ -102,6 +112,8 @@ object OSGi {
     Seq("akka.persistence.*"),
     imports = Seq(optionalResolution("org.fusesource.leveldbjni.*"), optionalResolution("org.iq80.leveldb.*")))
 
+  val persistenceTyped = exports(Seq("akka.persistence.typed.*"))
```

Comment:
isn't this needed for the other -typed modules also?
* actorTestkitTyped
* clusterTyped
* clusterShardingTyped
* streamTyped

Reply:
Hi, 
I have just added what I actually required. In principle I would agree except for the testkit unless someone wanted to use the testkit inside an OSGi enabled container. 

With respect to the others it might be better to have some kind of validation to make sure that the jar file contains what you expect. I might be able to invest some time in a raw validity check that is better than nothing next week. I am on site all week this week. 

Rgds
Andreas